html-forms/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
```

Comment:
You want to use one form element here, not a form element for each input field.  Stick to one, and just have your input fields as children for you r form.